deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
```

Comment:
Just checked and this may have been an issue on my local -- I must have stopped the provisioning job on my VM before it had created the MMW user, but it's there now that I've fully provisioned the thing.

I'm going to add `setuid mmw` back